MultiSlider.js

```diff
@@ -432,23 +433,48 @@ export default class MultiSlider extends React.Component {
     const body = (
       <React.Fragment>
         <View style={[styles.fullTrack, { width: sliderLength }]}>
-          <View
-            style={[
-              styles.track,
-              this.props.trackStyle,
-              trackOneStyle,
-              { width: trackOneLength },
-            ]}
-          />
-          <View
-            style={[
-              styles.track,
-              this.props.trackStyle,
-              trackTwoStyle,
-              { width: trackTwoLength },
-            ]}
-            {...(twoMarkers ? this._panResponderBetween.panHandlers : {})}
-          />
+          {renderTrack ? (
+            renderTrack({
+              selectedStyle: !twoMarkers,
```

Comment:
Also, what's the point of passing down selectedStyle if we're going to render it outside?

Reply:
I don't fully understand the library so I just re-used some code I found around the component to see if the track need the selected style or the normal style, this is the code I as reference:

```
const trackOneStyle = twoMarkers
      ? unselectedStyle
      : selectedStyle || styles.selectedTrack;
const trackTwoStyle = twoMarkers
      ? selectedStyle || styles.selectedTrack
      : unselectedStyle;
```

there is only one render function for all the tracks, so in order to know which one need the selected style I need to pass back the value, then on the app there is this code:
```
renderTrack={({ style, selectedStyle }) =>
    selectedStyle ? (
    <LinearGradient
        start={{ x: 0, y: 0 }}
        end={{ x: 1, y: 1 }}
        colors={['#C7D300', '#08AF97']}
        style={{
        ...style,
        ...styles.trackGradient,
        }}
    />
    ) : (
    <View style={style} />
    )
}
```

I think it could be written in a better way but as I don't want to break anything I tried to keep the changes as much back compatible as possible.

I think that with 1 marker there are 2 different tracks, sliding up/down will change the width of the 2 tracks, with 3 marker we'll have 3 tracks, 1 on the left, one in the center and one at the right, but I didn't spent too much time trying to understand the slider